src/main/scala/com/fulcrumgenomics/vcf/MakeTwoSampleMixtureVcf.scala

```diff
@@ -50,14 +47,19 @@ object MakeTwoSampleMixtureVcf {
 
   val TumorSampleName  = "tumor"
   val NormalSampleName = "normal"
+
+  val HeaderLines = MakeMixtureVcf.HeaderLines ++ Seq(
+    new VCFFilterHeaderLine(GermlineFilter,  "Evidence seen in the normal sample"),
+    new VCFFilterHeaderLine(MultiAllelicFilter, "Locus is multi-allelic in chosen samples.")
+  )
 }
 
 /**
   * Creates a VCF by mixing two germline samples at a given proportion.
   *
   * @author Tim Fennell
   */
-@clp(group=ClpGroups.VcfOrBcf, description=
+@sopt.clp(group=ClpGroups.VcfOrBcf, description=
```

Comment:
This is what keeps happening:
![useless](https://cloud.githubusercontent.com/assets/1609210/22482405/d38c1f44-e7c6-11e6-9f04-dceeb8011cad.gif)